lib/index.js

```diff
@@ -14,6 +14,15 @@ const runTasks = async (opts, di) => {
     Object.assign(container, di);
     container.config = container.config || new Config(opts);
 
+    if ('extends' in container.config.options) {
```

Comment:
Perhaps `typeof 'string'` makes sense, so you can set it to `false` or `undefined` as well to disable without issues (there are types etc. but they're not always used/adhered to).

Reply:
I like that! I will update it alone the tests :)

Reply:
Updated with tests :)

Reply:
Updated with tests :)